pages/about.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="/stylesheets/style.css" rel="stylesheet">
```

Comment:
Careful with your file paths! The `/stylesheets/style.css` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `./stylesheets/style.css`. The dot at the beginning signals to start searching in the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/